yaksh/selenium_test/selenium_test.py

```diff
@@ -0,0 +1,141 @@
+from selenium import webdriver
```

Comment:
This can be just added to the tests directory instead of creating a new `selenium_test`.